lib/jsoncpp/json/UPDATING

```diff
@@ -1,16 +1,7 @@
 #!/bin/sh
 cd ..
-svn co https://jsoncpp.svn.sourceforge.net/svnroot/jsoncpp/trunk/jsoncpp jsoncpp
-svn up jsoncpp
+git clone https://github.com/open-source-parsers/jsoncpp -b 1.8.3
```

Comment:
We don't need the entire history of this library, do we?

Reply:
i don't know i just updated this file, what option do you want me to add ?

Reply:
`--depth 1`, but that's a minor request, as this update script will be only executed once per year, if even.